server/lobbyconnection.py

```diff
@@ -16,12 +16,16 @@
 import server.metrics as metrics
 from server.db import FAFDatabase
 from sqlalchemy import and_, func, select, text
+from sqlalchemy.sql.functions import now as sql_now
```

Comment:
From sqlalchemy import func, `func.NOW(...)`
Func is proxy, you can all with it any db function.

Reply:
Ah, that's nice, of course!